### PR TITLE
Update instruction for new version of eval runtime-api

### PR DESCRIPTION
**Pull request by xingyaoww**
- Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

**End-user friendly description of the problem this fixes or functionality that this introduces**

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

- Update instruction for new version of (hopefully wayyy more stable) eval runtime-api

---
**Link of any specific issues this addresses**

### CODE REVIEW

**Review by enyst — APPROVED** [review]
Ah! Thanks! Can't believe I forgot again about the custom AppConfig. I just about to look into why it didn't get set automatically from env 😅 

Let's do this and I'll give it a try.

File: evaluation/swe_bench/README.md [hunk 1]
```diff
@@ -163,7 +163,8 @@ This is in limited beta. Contact Xingyao over slack if you want to try this out!
 
 ```bash
 # ./evaluation/swe_bench/scripts/eval_infer_remote.sh [output.jsonl filepath] [num_workers]
-ALLHANDS_API_KEY="YOUR-API-KEY" RUNTIME=remote EVAL_DOCKER_IMAGE_PREFIX="us-docker.pkg.dev/evaluation-428620/swe-bench-images" evaluation/swe_bench/scripts/eval_infer_remote.sh evaluation/outputs/swe_bench_lite/CodeActAgent/Llama-3.1-70B-Instruct-Turbo_maxiter_30_N_v1.9-no-hint/output.jsonl 16 "princeton-nlp/SWE-bench_Lite" "test"
+ALLHANDS_API_KEY="YOUR-API-KEY" RUNTIME=remote SANDBOX_REMOTE_RUNTIME_API_URL="https://runtime.eval.all-hands.dev" EVAL_DOCKER_IMAGE_PREFIX="us-central1-docker.pkg.dev/evaluation-092424/swe-bench-images" \
+evaluation/swe_bench/scripts/eval_infer_remote.sh evaluation/outputs/swe_bench_lite/CodeActAgent/Llama-3.1-70B-Instruct-Turbo_maxiter_30_N_v1.9-no-hint/output.jsonl 16 "princeton-nlp/SWE-bench_Lite" "test"
```
Comment (mamoodi) [inline evaluation/swe_bench/README.md:167]: The output I have is:
evaluation/evaluation_outputs/outputs/swe-bench-lite/CodeActAgent/....

Comment (xingyaoww) [reply evaluation/swe_bench/README.md:167]: Good catch!

Comment (xingyaoww) [inline evaluation/swe_bench/README.md:167]:
```suggestion
evaluation/swe_bench/scripts/eval_infer_remote.sh evaluation/evaluation_outputs/outputs/swe_bench_lite/CodeActAgent/Llama-3.1-70B-Instruct-Turbo_maxiter_30_N_v1.9-no-hint/output.jsonl 16 "princeton-nlp/SWE-bench_Lite" "test"
```

**Pull request merged**